src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs

```diff
@@ -325,7 +326,15 @@ public IEnumerable<OneWayCompatibilityMappingEntry> CompatibilityMappings
                             new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.Windows, FrameworkConstants.MaxVersion)),
                             new FrameworkRange(
                                 new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.WinRT, FrameworkConstants.EmptyVersion),
-                                new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.WinRT, new Version(4, 5, 0, 0))))
+                                new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.WinRT, new Version(4, 5, 0, 0)))),
+
+                        // NetCoreApp projects support NetStandard
+                        new OneWayCompatibilityMappingEntry(new FrameworkRange(
+                            new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, new Version(1, 0, 0, 0)),
+                            new NuGetFramework(FrameworkConstants.FrameworkIdentifiers.NetCoreApp, FrameworkConstants.MaxVersion)),
+                            new FrameworkRange(
+                                FrameworkConstants.CommonFrameworks.NetStandard10,
```

Comment:
Makes since I guess I was expecting to see it lower similar to where we have UAP mapped but I guess it was there because it maps both dotnet and netstandard.